code/modules/client/preference/preferences_toggles.dm

```diff
@@ -201,6 +201,19 @@
 		to_chat(src, "You will no longer hear musical instruments.")
 	feedback_add_details("admin_verb","TInstru") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
 
+/client/verb/Toggle_disco() //to toggle off heartbeat sounds, in case they get too annoying
```

Comment:
nice comment :D

Reply:
That comes from copy-pasting, but these toggles are made for that.

Reply:
My heart beats to the bass anyway, same deal